### PR TITLE
Fixes #244

### LINKED ISSUE #244: As a developer I want information on Sprint 3 release to be uploaded to the wiki
**Is your feature request related to a problem? Please describe.**
All other releases have been documented on the wiki.

**Describe the solution you'd like**
Information should be added to https://github.com/aau-giraf/wiki/blob/master/docs/releases/index.md and a new file should be added to https://github.com/aau-giraf/wiki/tree/master/docs/releases.

### PR DESCRIPTION
**Pull request by Nannanas**
Made new 2020s3r1 file. Updated index.md to include this new file. Deleted a row in 2020s2r1.

fixes #244

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw607f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw608f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by rasmus91 — APPROVED** [review]

**Review by elaurs — APPROVED** [review]

File: docs/releases/2020s3r1.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+**Release**
+
+These are the user stories that where included in 2020s3r1:
```
Comment (Badgie) [inline docs/releases/2020s3r1.md:3]: 'where' should be 'were'

File: docs/releases/index.md [hunk 2]
```diff
@@ -11,4 +11,5 @@ The following pages in this section describes the overall changes made for each
 | [2019S3R1](2019s3r1.md)    | Friday 26-04-2019|
 | [2020S1R1](2020s1r1.md)    | Friday 03-04-2020|
 | [2020S2R1](2020s2r1.md)    | Friday 01-05-2020|
+| [2020S3R1](2020s3r1.md)    | Friday 19-05-2020|
```
Comment (Badgie) [inline docs/releases/index.md:14]: Today is Tuesday.

**Review by Badgie — CHANGES_REQUESTED** [review]

**Pull request merged**